org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/IndexOnlyProjectTest.xtend

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2017 TypeFox GmbH (http://www.typefox.io) and others.
```

Comment:
wrong year

Reply:
The copyright should be from 2019 onwards for enfore